README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+
+
+### My biography
+
+My name is **Guillermo Guevara** and I am second year MPP at Harris. Before coming to Harris, I held several positions at the Mexican Federal Government.I am interested in:
+
+- Policing
+- Crime prevention
+- Political Accountability
+
+[Here you can check a photo of my idol Andy Murray](https://en.wikipedia.org/wiki/Andy_Murray#/media/File:Andy_Murray_(44087043305).jpg)
```

Comment:
This is a great use of a URL to render an image! Make sure you can also render images from local paths -- it's a great way to learn about directories and file systems at least.